test/core/annotations/test_before_after_test_annotation.pkb

```diff
@@ -0,0 +1,138 @@
+create or replace package body test_before_after_annotations is
+  g_tests_results clob;
+
+  procedure create_tests_results is
+    pragma autonomous_transaction;
+
+    l_test_package_spec  varchar2(32737);
+    l_test_package_body  varchar2(32737);
+    l_dummy_utility_pkg_spec varchar2(32737);
+    l_dummy_utility_pkg_body varchar2(32737);
+    l_test_results  ut3.ut_varchar2_list;
+    
+    procedure drop_package(a_package_name varchar2)
+    is
+    begin
+      execute immediate 'drop package '||a_package_name;
+    end;
+  begin
+    l_test_package_spec := '
+        create or replace package dummy_before_after_test is
+            --%suite(Package to test annotations beforetest and aftertest)
+            
+            l_dummy_1 integer;
```

Comment:
I would put those variables into `dummy_utility_pkg_1`.
That way we avoid bidirectional dependency where test package depending on utility and vice versa.